api/v1beta1/flowcollector_types.go

```diff
@@ -436,6 +436,11 @@ type FlowCollectorFLP struct {
 	// `conversationTerminatingTimeout` is the time to wait from detected FIN flag to end a conversation. Only relevant for TCP flows.
 	ConversationTerminatingTimeout *metav1.Duration `json:"conversationTerminatingTimeout,omitempty"`
 
+	//+kubebuilder:default:="defaultCluster"
+	// +optional
+	// `clusterName` is the name of the cluster.
+	ClusterName string `json:"clusterName,omitempty"`
+
```

Comment:
For Openshift we can automatically retreive `ClusterID` from api
https://pkg.go.dev/github.com/openshift/api@v0.0.0-20230707160225-81d582da354b/config/v1#ClusterID

https://access.redhat.com/solutions/5280291

Reply:
```suggestion
	//+kubebuilder:default:=""
	// +optional
	// `clusterName` is the name of the cluster.
	ClusterName string `json:"clusterName,omitempty"`

```
I would suggest to set default as empty to avoid FLP extra stage for single cluster usage. WDYT ?

Reply:
If we automatically retrieve clusterID from openshift when clusterName is blank, we really cannot distinguish between single cluster vs multi-cluster. To distinguish between them, we would have to add an explicit parameter to the flowcollector config file.

Reply:
Default set to empty.